javascript/ql/src/Statements/UselessRangeCheck.qhelp

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+If a condition always evaluates to true or always evaluates to false, this often indicates
+incomplete code or a latent bug and should be examined carefully.
+</p>
+
+</overview>
+<recommendation>
+
+<p>
+Examine the surrounding code to determine why the condition is useless. If it is no
```

Comment:
Here too, perhaps replace "useless" (I leave it up to you. It doesn't sound so bad here)